### DISCUSSION TITLE
SPITFIRE options for FATES

### DISCUSSION THREAD

**Discussion by ychengen** [ESCOMP/CTSM#1943] [Q&A] [answered]
Hi,

CTSM-FATES provides different options from an unset value to 0-5 for using SPITFIRE. If I configure my run with "_I2000Clm51FatesRs_" and set "_fates_spitfire_mode = 0_" in _user_nl_clm_, is there no fire at all during my simulation? Or does my simulation still include/inherit fires from the host land model if I don't specify the "_-nofire_" flag when setting up my case? Thank you!

**Comment by jkshuman** [comment, marked as answer]
@ychengen by default there is no active fire for a case. The default value for the SPITFIRE mode is set with a namelist setting, so for clm in the user_nl_clm file: 
fates_spitfire_mode = 0

There will be no fire provided from the host land model. You will only have active fire by setting fates_spitfire_mode to something other than 0.

More details about the active fire modes can be found in the FATES User's Guide https://fates-users-guide.readthedocs.io/en/latest/user/SPITFIRE-Namelist-Options.html

**Reply by ychengen** [reply]
@jkshuman Thanks for clarifying this.

I am confused about these options since I have an impression that I was told during an online CESM workshop previously that setting fates_spitfire_mode = 0 would still have fires from CLM5 Fang Li model, if "_-nofire_" flag is not specifically given when compiling the model.

Thanks again for your answer.

**Reply by jkshuman** [reply]
@ychengen the CLM5 Li fire model is not available as part of FATES as it is not configured to run with the size-structured vegetation of FATES. The Li fire model is available as part of CLM5 BGC, but SPITFIRE is not configured to run in CLM5-BGC.

**Comment by ekluzek** [comment]
Closing as answered.
